README.md

```diff
@@ -272,45 +273,45 @@ application.
 
 3. Update packages using pacman:
 
-	```$ pacman -Syu```
+	`$ pacman -Syu`
 
 4. Exit the MSYS shell using Alt+F4 or by clicking X at top-right corner. It is Very Important to do not exit to shell!!.
 
 5. Start `MSYS2 MINGW64` from Menu Start
 
 6. Update packages again using pacman:
 
-	```$ pacman -Syu```
+	`$ pacman -Syu`
 
 7. Install dependencies:
 
     To build for 64-bit Windows:
-	
-    ```$ pacman -S git mingw-w64-x86_64-toolchain make mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-openssl mingw-w64-x86_64-libsodium mingw-w64-x86_64-hidapi automake autoconf binutils patch```
+
+    `$ pacman -S git mingw-w64-x86_64-toolchain make mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-openssl mingw-w64-x86_64-libsodium mingw-w64-x86_64-hidapi automake autoconf binutils patch`
 
 **Building**
 
 * Download Arqma with command:
 
-	```$ git clone https://github.com/arqma/arqma```
+	`$ git clone https://github.com/arqma/arqma`
 
 * Change branch to last Release:
 
-	```$ cd arqma && git checkout release-v0.6.0```
+	`$ cd arqma && git checkout release-v0.6.0`
 
 * Activate and update submodules:
 
-        ```$ git submodule init && git submodule update```
+  `$ git submodule init && git submodule update`
 
 * If you are on a 64-bit system, run:
 
-        ```$ make release-static-win```
+  `$ make release-static-win`
 
 * The resulting executables can be found in `build/release/bin`
```

Comment:
Should we not update that line as well?
Current statement is valid only when use USE_SINGLE_BUILDDIR=1 within ${MAKE} command. 
Use [Makefile](Makefile) for reference.

Reply:
Nope its correct

Reply:
I do know about that. But that statement is not true without USE_SINGLE_BUILDDIR=1
And it is not repated to platform you are building binaries at. It is the same folder structure at every one. It is related to github_cli which creating folders related to branch you are building. At master branch it will be: build/<platform>/master/release/bin .

Reply:
Corrected